framework/src/main/java/org/tron/core/services/interfaceOnPBFT/http/PBFT/GetTransactionByIdOnPBFTServlet.java

```diff
@@ -0,0 +1,26 @@
+package org.tron.core.services.interfaceOnPBFT.http.PBFT;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.tron.core.services.http.GetTransactionByIdServlet;
+import org.tron.core.services.interfaceOnPBFT.WalletOnPBFT;
+
+@Component
+@Slf4j(topic = "API")
+public class GetTransactionByIdOnPBFTServlet
+    extends GetTransactionByIdServlet {
+
+  @Autowired
+  private WalletOnPBFT walletOnPBFT;
+
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+    walletOnPBFT.futureGet(() -> super.doGet(request, response));
+  }
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response) {
+    walletOnPBFT.futureGet(() -> super.doPost(request, response));
+  }
+}
```

Comment:
EOF

Reply:
Hadoop‘s source file also has not the EOF. It is unnecessary.

Reply:
> EOF

Hadoop‘s source file also has not the EOF. It is unnecessary.

Reply:
ref: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
> ref: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Java code style does not have this requirement.

Reply:
Then, how will you explain that your Github diff page & git diff command display it as a warning style?